python/ray/data/_internal/stats.py

```diff
@@ -774,6 +774,13 @@ def to_string(self) -> str:
             out += (
                 "* Batch iteration time breakdown (summed across prefetch threads):\n"
             )
+            if self.wait_time.get():
+                out += "    * In block fetching: {} min, {} max, {} avg, {} total\n".format(
```

Comment:
@amogkam - do you think it would be clear if we show "block fetching" time here? This is to fetch the block from other node to the consumer node.

Reply:
what does that actually mean though?

the actual time is more just like "task launching overhead" right? since `ray.wait` is non-blocking?

block fetching time is measured more by the `ray.get`